cirq-core/cirq/contrib/qasm_import/_parser.py

```diff
@@ -315,13 +322,19 @@ def p_circuit_empty(self, p):
 
     def p_new_reg(self, p):
         """new_reg : QREG ID '[' NATURAL_NUMBER ']' ';'
-        | CREG ID '[' NATURAL_NUMBER ']' ';'"""
-        name, length = p[2], p[4]
+        | QUBIT '[' NATURAL_NUMBER ']' ID ';'
```

Comment:
The OpenQASM 3 seems to allow [scalar qubits](https://openqasm.com/versions/3.0/language/types.html#qubits) defined as `qubit some_name;`.

Would it be feasible to handle it here?

Reply:
Added scalar qubits (and bits)